### PR TITLE
:sparkles: Contact Page

### PR DESCRIPTION
**Pull request by iamrohitsuthar**
Contact Page

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -49,6 +49,9 @@ This files contents are outlined below.
     color: #FFF
 }
 
+.heart {
```
Comment (saglaniharsh) [inline assets/css/style.css]: No need to do this use `<i class="fa fa-heart"></>`

File: assets/css/style.css [hunk 2]
```diff
@@ -97,6 +100,10 @@ padding: 18px 50px;
     color: #fff !important;
   }
    
+.btn-primary:hover {
```
Comment (saglaniharsh) [inline assets/css/style.css:100]: effect is good. try to change color. Pink is does not look that good

File: contact.html [hunk 3]
```diff
@@ -1,290 +1,239 @@
...
-      <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i,800,800i&amp;subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese" rel="stylesheet">
-   </head>
-   <body>
-      <div class="bg-all">
+
+<head>
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <title>Contact Page </title>
```
Comment (saglaniharsh) [inline contact.html]: Change to "Contact Us"

File: contact.html [hunk 4]
```diff
@@ -1,290 +1,239 @@
...
+                              <a href="about.html" class="">About</a>
+                           </li>
+                           <li class="dropdown submenu">
+                              <a href="services.html" class=""> Services</a>
+                           </li>
+                           <li class="dropdown submenu">
+                              <a href="blog.html" class="">Blog</a>
+                           </li>
+                           <li class="dropdown submenu">
+                              <a href="elements.html" class="">Elements</a>
```
Comment (saglaniharsh) [inline contact.html]: Remove this or copy nav from #1

File: contact.html [hunk 5]
```diff
@@ -1,290 +1,239 @@
...
+            <!-- End Header_Area -->
+            <!-- #banner start -->
+            <section id="banner" class="pt-0">
+               <div class="container ">
+                  <div class="row wow fadeInUp">
+                     <!-- #banner-text start -->
+                     <div id="banner-text" class="col-md-12 text-c text-center ">
+                        <h5 class="wow fadeInUp main-h" data-wow-delay="0.2s">Contact Us</h5>
+                        <p class="banner-text wow fadeInUp main-h3" data-wow-delay="0.8s">No hours sank into aggregating
+                           and cleaning data. No complex SQL queries required. Just the answers <br> teams need to make
```
Comment (saglaniharsh) [inline contact.html]: copy description from #1 too

File: contact.html [hunk 6]
```diff
@@ -1,290 +1,239 @@
...
-                        <div class="top-banner wow fadeInRight text-left" style="visibility: visible; animation-name: fadeInRight;">
-                           <a id="#services" href="contact.html" class="btn btn-primary wow fadeInUp  js-scroll-trigger" data-wow-delay="1s" style="visibility: visible; animation-delay: 1s; animation-name: fadeInUp;">CONTACT SALES</a>
+                           <div class="contact-info-section">
+                              <h4>Available 24x7x365</h4>
+                              <ul class="list-style-1 no-margin-bottom">
+                                 <li>
+                                    <p><i class="fa fa-globe text-center"></i> <strong>Country:</strong> India</p>
+                                 </li>
+                                 <li>
+                                    <p><i class="fa fa-phone text-center"></i> <strong>Phone:</strong> (+91) 1234 5678
```
Comment (saglaniharsh) [inline contact.html]: Let's use a real phone number

File: contact.html [hunk 7]
```diff
@@ -1,290 +1,239 @@
...
+                                       99</p>
+                                 </li>
+                                 <li>
+                                    <p><i class="fa fa-envelope text-center"></i> <strong>Email:</strong> <a
+                                          href="javascript:void(0)"
+                                          class="email_color_site">info@codinganddecoding.com</a></p>
```
Comment (saglaniharsh) [inline contact.html]: and also a our real email until we get paid hosting with the domain and custom email

**Review by saglaniharsh — CHANGES_REQUESTED** [review]

**Comment by saglaniharsh** [comment]
Also, from next time please make sure to select project on the right side while submitting pr

**Comment by iamrohitsuthar** [comment]
Okay, Done with the requested changes.
Now please check!

**Review by saglaniharsh — APPROVED** [review]

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
@iamrohitsuthar Just one more thing. Please update the footer with what's in #1 otherwise it will be overwritten when the branch is merged. Just keep the social icons and use rest from #1 
Thanks.

**Comment by iamrohitsuthar** [comment]
okay

**Comment by iamrohitsuthar** [comment]
Now please check

**Review by saglaniharsh — APPROVED** [review]

**Pull request merged**